api/src/main/scala/hmda/api/http/LarHttpApi.scala

```diff
@@ -83,10 +83,10 @@ trait LarHttpApi extends LarProtocol with ValidationResultProtocol with HmdaCust
   def validateRoute(lar: LoanApplicationRegister, checkType: String): scaladsl.server.Route = {
     val larValidation = system.actorSelection("/user/larValidation")
     val checkMessage = checkType match {
-      case "syntactical" => CheckSyntactical(lar, ValidationContext(None))
-      case "validity" => CheckValidity(lar, ValidationContext(None))
-      case "quality" => CheckQuality(lar, ValidationContext(None))
-      case _ => CheckAll(lar, ValidationContext(None))
+      case "syntactical" => CheckSyntactical(lar, ValidationContext(None, None))
+      case "validity" => CheckValidity(lar, ValidationContext(None, None))
+      case "quality" => CheckQuality(lar, ValidationContext(None, None))
+      case _ => CheckAll(lar, ValidationContext(None, None))
```

Comment:
Would it make sense to say `Some(2017)` for this context? I'm open to arguments for `None`. Just want to make sure it's a conscious decision.

Related: what is the intended behavior of edits when there's not a year? Are they just skipped? (Or, should that situation be impossible?)